kafka-ui-react-app/src/components/common/PageLoader/PageLoader.tsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import * as SP from 'components/common/Spinner/Spinner.styled';
 
 import * as S from './PageLoader.styled';
 
 const PageLoader: React.FC = () => (
   <S.Wrapper>
-    <S.Spinner role="progressbar" />
+    <SP.Spinner
+      role="progressbar"
+      width={80}
+      height={80}
+      borderWidth={10}
+      emptyBorderColor={false}
```

Comment:
do we really need to customize all this props here?

Reply:
I think yes, but we can create default props object and customize on demand. In PageLoader it's gonna be empty in such case

Reply:
Updated in https://github.com/provectus/kafka-ui/pull/3551/commits/8a6706f0dd3c26aa2fc5f6bcd0ddf91625bf1092